dev-workflow-ui/src/ch/ivyteam/workflowui/tasks/TasksDetailsIvyDevWfBean.java

```diff
@@ -158,6 +159,23 @@ private boolean canResume() {
     return isActivator || isWorker;
   }
 
+  public boolean showInfoBanner() {
+    return selectedTask.getState() == TaskState.RESUMED || selectedTask.getState() == TaskState.DONE;
+  }
+
+  public String getInfoBannerSeverity() {
```

Comment:
user can be null if no user is logged in.
Only show a warning if you are working on a task in a different session.
1. Task in state TaskState.RESUMED, TaskState.CREATED
2. Current session is activator
3. task.getWorkerSession != current session